apps/widget-configurator/src/app/configurator/consts.ts

```diff
@@ -3,6 +3,7 @@ import { TradeType, TokenInfo } from '@cowprotocol/widget-lib'
 
 import { TokenListItem } from './types'
 
+export const DEFAULT_PARTNER_FEE_RECIPIENT = '0x8BFE784432C91169Bdf3469B52ac589912dE21C3'
```

Comment:
Maybe add `0xcA771eda0c70aA7d053aB1B25004559B918FE662` which is CoW Dao

Reply:
Changed